docs/changelog.md

```diff
@@ -1,7 +1,8 @@
 # TBD
+### Changes
+* Upgrade to core to 1.58.1 and engine to 1.32.1
```

Comment:
Is this a breaking change, or is it okay?

Reply:
it is not a breaking change because users could use this module version without changing anything